feg/gateway/utils/configs.go

```diff
@@ -117,3 +117,20 @@ func getFlagValue(flagName string) string {
 	}
 	return res
 }
+
+func PrintBuildInfo() string {
+	delimiter := "********"
+	return fmt.Sprintf("%s\n"+
+		"Build Info\n"+
+		"\tCommit Branch: %s\n"+
+		"\tCommit Tag: %s\n"+
+		"\tCommit Hash: %s\n"+
+		"\tCommit Date: %s\n"+
+		"%s\n",
+		delimiter,
+		os.Getenv("MAGMA_BUILD_BRANCH"),
+		os.Getenv("MAGMA_BUILD_TAG"),
+		os.Getenv("MAGMA_BUILD_COMMIT_HASH"),
+		os.Getenv("MAGMA_BUILD_COMMIT_DATE"),
+		delimiter)
+}
```

Comment:
wouldn't it be easier & more reliable to embed all this information into Go binaries instead?

something like:

package main
...
var GitCommitHash string
...

and then buld with:

export GIT_COMMIT_HASH=$(git rev-list -1 HEAD) && go build -ldflags "-X main.GitCommitHash=$GIT_COMMIT_HASH" ...